packages/template-retail-react-app/app/components/check-email/index.jsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
```

Comment:
It might be worth while to add a block comment at the top here to describe the purpose of this page. I was initially confused that we have a page and a component with the same name "check-email". 

Speaking of name, I think there might be a better name out there than "check-email" as it would be nice to have a name that implies some kind of function, this is quite vague.